### DISCUSSION TITLE
我是meowjiao, 我的QQ被冻结了

### DISCUSSION THREAD

**Discussion by catmeowjiao** [ITCraftDevelopmentTeam/Forum#22] [General]
我是meowjiao, 我的主QQ号被冻结了, 请问IT Craft Development Team现在的群号是什么?
我在别的QQ上尝试搜索"701257458", 没有搜索结果.
谢谢!

**Comment by catmeowjiao** [comment]
草，发错地方了

**Reply by This-is-XiaoDeng** [reply]
Transfer了

**Comment by This-is-XiaoDeng** [comment]
如果要加入 IT Craft Development Team 的群聊，请先填写一份[申请](https://github.com/ITCraftDevelopmentTeam/Forum/issues/new/choose)，详见[此文章](https://github.com/orgs/ITCraftDevelopmentTeam/discussions/8)

等你哟！
